docs/pages/cloud/architecture.mdx

```diff
@@ -47,6 +47,25 @@ The Teleport [proxy service](https://goteleport.com/docs/architecture/proxy/) ca
 
 The multi-region option is currently opt-in by default. Once you have an account, please reach out to your account manager, customer success engineer, or support@goteleport.com. A future update will expand the region availability and make all regions available by default.
 
+## Releases
+
+Teleport Cloud customers are typically running the latest release of the
```

Comment:
Teleport Cloud only serves the latest stable release of the Teleport software for its customers.